spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -287,6 +287,7 @@ def main(argv: Sequence[str]):
             thr = (arguments.binarize_prediction if arguments.binarize_prediction
                    else models.MODELS[name_model]['thr'])  # Default `thr` value stored in model dict
             im_lst, target_lst = inference.segment_non_ivadomed(path_model, model_type, input_filenames, thr,
+                                                                keep_largest=arguments.keep_largest,
                                                                 use_gpu=use_gpu, remove_temp_files=arguments.r)
```

Comment:
@naga-karthik If you have some time, would you be able to add functionality for the other `sct_deepseg` postprocessing arguments, too? 

There are only two other arguments (`-fill-holes` and `-remove-small`), and you should be able to copy the ivadomed functions from [`postprocessing.py`](https://github.com/ivadomed/ivadomed/blob/1fccf77239985fc3be99161f9eb18c9470d65206/ivadomed/postprocessing.py) in much the same way as was done for `-largest`.

All you need to do is pass the argument value to `segment_non_ivadomed`, then call the necessary function if the value `is not None`.

Reply:
Sure! I added both these postprocessing options in the following commits: 
1. `-fill-holes` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4482/commits/2d43faa1c0f08332abd9b4368e9841bac3024c84  
2. `-remove-small` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4482/commits/07ec6d9ae60b37f758e37046408d56caf81c48f5

the fill-holes part took some time as the original logic was a bit complicated (it was calling a function tied to a class), the commit tries to include the whole logic into a single function `remove_small_objects`

Please let me know your thoughts on these changes!

Reply:
Thank you so much for taking the time to do this! Taking a look now. :hearts: